integration-tests/discovery-servicecomb-tests/price-provider/src/main/java/com/huaweicloud/sample/SchemaController.java

```diff
@@ -52,14 +52,13 @@ public String testSchemaGenerator() throws Exception {
     assertThat(schemaContents.size()).isEqualTo(schemas.size());
     assertThat(schemaContents.size()).isEqualTo(schemaSummary.size());
 
-    String a1 = schemaContents.get("schemaContentController");
-    String a2 = readFile("SchemaContentController.yaml");
-    OpenAPI swagger2 = Yaml.mapper().readValue(a2, OpenAPI.class);
-    OpenAPI swagger1 = Yaml.mapper().readValue(a1, OpenAPI.class);
-    if (swagger1.equals(swagger2)) {
+    String priceSchema = schemaContents.get("price");
+    OpenAPI swagger = Yaml.mapper().readValue(priceSchema, OpenAPI.class);
+    // 契约接口数为当前服务所有接口数，如果有接口增加，比较值相应改变
+    if (swagger.getPaths().size() == 43) {
       return "success";
     } else {
-      return a1;
+      return priceSchema;
```

Comment:
add a file and compare the file content so that we can check the generated swagger is correct.

Reply:
fixed